src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -415,6 +415,61 @@ private function getInlineFromAttributes(array $attributes)
         return $inline;
     }
 
+    private function getTableEditorPagedef($pagedef): string
+    {
+        $inputFilterUtil = $this->getInputFilterUtil();
+        $customTableEditor = $inputFilterUtil->getFilteredInput('sTableEditorPagdef');
```

Comment:
We could add the empty string as second argument, so that the comparison in the line below doesn't need to check for null.